tests/model_tests/test_contact.py

```diff
@@ -1,22 +1,40 @@
 import uuid
 
+#
 from core.db import db
-from core.db.entities import Contact
+from core.db.entities import Contact, Organisation
 
 
-def test_contact(app_ctx):
+def test_contact_organisation(app_ctx):
+    """
+    Test basic relationship structure b/w Contact and Organisation.
+
+    Sanity check that fk relationships are looked up as expected.
```

Comment:
Can we replace this with more [inclusive naming](https://inclusivenaming.org/word-lists/tier-2/sanity-check/) e.g. confidence check

Reply:
aha, yes that is a very good idea.